src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship.Api/Models/Applications/PostUpdateTrainingCourseApiRequest.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace SFA.DAS.FindAnApprenticeship.Api.Models.Applications;
+
+public class PostUpdateTrainingCourseApiRequest
+{
+    public Guid CandidateId { get; set; }
+    public string CourseName { get; set; }
+    public int YearAchieved { get; set; }
```

Comment:
maybe uint for this? might be worth decreasing the size of the field in the db as well to a short?